backups/burp.rst

```diff
@@ -0,0 +1,77 @@
+BURP - Backup And Resotre Program
+=================================
+
+What is BURP?
+-------------
+
+So what is BURP and why do we use it?
+
+BURP is a network backup and restore program. It attempts to reduce network traffic and the amount of space that is used by each backup. - burp.grke.org
+
+BURP basically makes a backup of the whole file system. This dumps are very neat, you can view or restore the whole backup or just a single file. 
+Currently the backups on are made with a lot of different scripts. 
+The standard procedure of the scripts is something like that:
+
+#. Compress a directory tree.
+
+#. Move the compressed directories to the backup server (rsync).
+
+#. Delete the compress files if they are older than x days.
+
+For every step we have a single script and sometimes there is a collections of script for every server.
+
+BURP combines all these steps in one Program and makes backups more comprehensible and easy to handle. 
+
+Search and restore a file
+-------------------------
+
+#. Show backups all available backups.
+
+   .. code:: 
+
+      sudo burp -a list 
+
+      Backup: 0000054 2017-10-21 23:08:28 +0200 (deletable)
+      Backup: 0000055 2017-10-22 22:33:40 +0200 (deletable)
+      ...
+
+#. Show content of a backup.
+
+   .. code::
+
+      sudo burp -a list -b 55
+
+      /var/spool/postfix/public/showq
+      /var/spool/postfix/saved
+      /var/spool/postfix/trace
+      /var/spool/postfix/usr
+      ...
+
+   .. note:: 
+      | the number 55 refers to the number of the backup in step 1.
+      | e.g. Backup: 0000055 2017-10-22 22:33:40 +0200 (deletable)
+
+#. Search for all files in backup that match the regex condition.
+
+   .. code::
+
+      sudo burp -a list -b 55 -r 'nice2'
+
+      Backup: 0000055 2017-10-22 22:33:40 +0200 (deletable)
+      With regex: nice2
+      /home/nils.kreienbuehl/dumps/nice2_ethz_dump_cloud.sql
+      /home/nils.kreienbuehl/dumps/nice2_heks.sql
+      /home/nils.kreienbuehl/dumps/nice2_heks2017-09-13T12:20:29Z.sql
```

Comment:
We should restore an actual backup here (`/var/lib/postgresql-backup/` on db2).

I'll open a ticket to make them available on db1.